source/libs/features.tsx

```diff
@@ -139,6 +126,13 @@ const getShortcuts = (): Shortcut[] => [...shortcutMap.values()];
  * Register a new feature
  */
 const add = async (definition: FeatureDetails): Promise<void> => {
+	// In chrome:// pages, just collect the features in a global variable
+	if (!location.protocol.startsWith('http')) {
+		window.collectFeatures = window.collectFeatures || new Map();
```

Comment:
Why not instantiate `collectFeatures` on `window` outside `add`?

https://github.com/sindresorhus/refined-github/pull/1780/files#diff-9f662d0cb609bc9bff7176ed8ee00bb6R29 is always throwing error on page load for me.

![image](https://user-images.githubusercontent.com/37769974/58177945-eb053f00-7cc2-11e9-8098-90c58623ba68.png)

Reply:
Same, just came to mention this.